docs/getting-started/installing-substrate.md

```diff
@@ -2,37 +2,44 @@
 title: "Installing Substrate"
 ---
 
+Following ways to install Substrate assume you are running Mac OS, Arch, or a Debian-based OS like Ubuntu. For Windows users, you can try it [here](https://github.com/paritytech/substrate#612-windows).
```

Comment:
```suggestion
Here are instructions to install Substrate for Mac OS, Arch, or a Debian-based OS like Ubuntu. For Windows users, you can try it [here](https://github.com/paritytech/substrate#612-windows).
```

We should probably port the Windows installation instructions to our docs.

Reply:
I haven't tested the Windows part, will add it later once fully test it.